BUILD

```diff
@@ -540,12 +540,100 @@ internal_gen_well_known_protos_java(
 )
 
 java_library(
+    name = "protobuf_java_lite",
```

Comment:
"java/core/src/main/java/com/google/protobuf" is a bad build package containing completely unrelated stuff (the path is specially crafted for Maven). Putting the target in the root BUILD file is much more cleaner.